lib/ruby_lsp/utils.rb

```diff
@@ -6,29 +6,73 @@ module RubyLsp
   VOID = T.let(Object.new.freeze, Object)
 
   # A notification to be sent to the client
-  class Notification < T::Struct
-    const :message, String
-    const :params, Object
+  class Notification
+    extend T::Sig
+
+    sig { returns(String) }
+    attr_reader :message
+
+    sig { returns(Object) }
+    attr_reader :params
+
+    sig { params(message: String, params: Object).void }
+    def initialize(message, params)
```

Comment:
Why not using named parameters so the change is transparent to the rest of the codebase?